### PR TITLE
qa_crowbarsetup: decrease top TO for HA compute (bsc#1024742)

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

**Comment by aplanas** [comment]
This PR is a workaround for the bug, injecting overrides for the pacemaker_remote service

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -5028,6 +5028,21 @@ function onadmin_run_cct
...
+        # overrides to compute nodes the belong to the pacemaker
+        # remote cluster. This override will decrease the
+        # TimeoutStopSec from 30min to 30secs, making the boot process
+        # faster.
+        if iscloudver 7plus; then
+            local remotes=$(crowbarctl proposal show \
+                --filter deployment.pacemaker.elements.pacemaker-remote \
+                --plain pacemaker services | cut -d' ' -f2)
+            local override=/etc/systemd/system/pacemaker_remote.d/override.conf
+            for node in $remotes; do
```
Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: missing: `local node`

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -5028,6 +5028,21 @@ function onadmin_run_cct
...
+                ssh $node "echo TimeoutStopSec=30s > $override; systemctl daemon-reload"
```
Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: Why is a systemd deamon-reload needed instead of reloading pacemaker itself?

Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: According to documentation when you change a service or override, you need to rescan with this command

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (aplanas) [inline scripts/qa_crowbarsetup.sh]: done

**Comment by aplanas** [comment]
I wonder if this still makes sense

**Pull request closed without merging**